scripts/xnat/check_new_sessions

```diff
@@ -205,7 +205,7 @@ def label_check(label):
         if match:
             return match
 
-def incomplete_scan_check( eid, manufacturer):
+def incomplete_scan_check( eid, manufacturer,label):
```

Comment:
@abonil91 small thing, but no space after the `(` and include a space after the comma 

`def incomplete_scan_check(eid, manufacturer, label):`